src/mapping.hpp

```diff
@@ -1,198 +1,12 @@
-)";
\ No newline at end of file
+extern const std::string keystonesJson;
```

Comment:
note: I might put a namespace around these files so that when mapping.hpp is included, it becomes clear what it was included for (you can search the token `mapping` in the file and find what's being use).

On the note of naming, these are very subjective, but...

It wasn't clear to me at first where these names were coming from. Partly because they don't follow the typical naming conventions for constants, usually either UPPER_SNAKE_CASE, or Hungarian camelCase with a leading k for "[k]onstant", like `kCamelCaseConstant`.

Things like that make it easy to see whether you're bringing in a type name for a class or a constant, or a function, or sometimes something else. It's a good idea to have different naming conventions for different types of things. There's some well established standards there, but every project has opinions about which naming standard is best.

Folks who've been around for a while will work with what a codebase has, but it's a lot easier for developers to come into a codebase and not make a mess when whatever the system is, it's clear and consistent. When it's not clear and consistent, there's a good chance I'll unintentionally do something that's pretty close but more familiar to me.

I'd make the case that `camelCase` is useful for function and method names, but it gets a little bit confusing when class names aren't spelled with `PascalCase` (like `camelCase` but with a leading capital).

Reply:
Yea I completely forgot about the differing naming scheme usually used by constants. MB. Ill probably run with UPPER_SNAKE_CASE for constants since thats just what I am used to. I probably also need to add a section to the "contributing" section of the readme detailing the naming schemes used for certain stuff.